engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/details/subscription-content-hosts.controller.js

```diff
@@ -4,42 +4,45 @@
  *
  * @requires $scope
  * @requires translate
- * @requires Subscription
+ * @requires Nutupane
+ * @requires Host
  * @requires ContentHostsHelper
  *
  * @description
  *   Provides the functionality for the subscription details for content hosts.
  */
 angular.module('Bastion.subscriptions').controller('SubscriptionContentHostsController',
-    ['$scope', 'translate', 'Subscription', 'ContentHostsHelper',
-    function ($scope, translate, Subscription, ContentHostsHelper) {
+    ['$scope', 'translate', 'Nutupane', 'Subscription', 'Host', 'ContentHostsHelper', 'CurrentOrganization',
+    function ($scope, translate, Nutupane, Subscription, Host, ContentHostsHelper, CurrentOrganization) {
+
+        var nutupane, params;
+
+        params = {
+          'organization_id': CurrentOrganization,
+          'sort_by': 'name',
+          'sort_order': 'ASC'
+        };
+
+        nutupane = new Nutupane(Host, params);
+        $scope.nutupane = nutupane;
+        $scope.table = nutupane.table;
+
+        $scope.controllerName = 'SubscriptionHosts';
+        $scope.successMessages = [];
+        $scope.errorMessages = [];
```

Comment:
I don't think this page actually needs success/error messages, I just copied and pasted that from another controller. I'll remove them